src/includes/configuration/before-send/ruby.mdx

```diff
@@ -4,7 +4,7 @@ Sentry.init do |config|
   config.before_send = lambda do |event, hint|
     # note1: if you have config.async configured, the event here will be a Hash instead of an Event object
     # note2: the code below is just an example, you should adjust the logic based on your needs
-    event.request.data = filter.filter(event.request.data)
+    event&.request&.data = filter.filter(event.request.data)
```

Comment:
I don't think the event would ever be `nil` 🤔 

also, I think we can be more explicit here

```ruby
if request = event.request
  request.data = filter.filter(request.data)
end

event
```

Reply:
Yeah that's true, event should always be there. Pushed a commit almost identical do this but didn't put the assignment in the conditional.